examples/dmd/parametric_tw_csv.py

```diff
@@ -0,0 +1,891 @@
+#  ******************************************************************************
+#  *
+#  * Copyright (c) 2013-2023, Lawrence Livermore National Security, LLC
+#  * and other libROM project developers. See the top-level COPYRIGHT
+#  * file for details.
+#  *
+#  * SPDX-License-Identifier: (Apache-2.0 OR MIT)
+#  *
+#  *****************************************************************************
+# // Compile with: make parametric_tw_csv
+# //
+# // Generate CSV or HDF database on heat conduction with either
+# // heat_conduction_csv.sh or heat_conduction_hdf.sh (HDF is more efficient).
+# //
+# // =============================================================================
+# //
+# // Parametric serial DMD command (for HDF version, append -hdf):
+# //   python3 parametric_tw_csv.py -o hc_parametric_serial -rdim 16 -dtc 0.01 -offline
+# //   python3 parametric_tw_csv.py -o hc_parametric_serial -rdim 16 -dtc 0.01 -online
```

Comment:
Without -list directory specified, how does it know which parameter to test?

Reply:
I don't exactly know the full workflow, but it does have `-list` option, which is set to `dmd_list` by default. `heat_conduction_csv.sh` (or hdf) uses the default setting, so the command does not need to specify `-list`.